package.json

```diff
@@ -21,30 +21,32 @@
     "test": "jest --coverage ./src"
   },
   "dependencies": {
-    "prop-types": "^15.6.2",
-    "react": "^16.7.0"
+    "apollo-client": "^2.6.3",
+    "graphql": "^14.4.2",
```

Comment:
didn't we have the `graphql` dependency nor `apollo-client` before?

Reply:
Nope. But that is not important. I just wanted to get rid of the peer dependency warnings